Rebus.Idempotency.MySql/Rebus.Idempotency.MySql.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <VersionPrefix>1.3.0</VersionPrefix>
+    <VersionPrefix>1.3.1</VersionPrefix>
```

Comment:
Given that the column specification is again different this feels like at least a minor number (if not major)

Reply:
@dietermijle I have pushed these packages before a couple of weeks back when Bjorn was on leave that friday, to try and move things forward. But yeah we are the only ones looking into it now so it's fine I guess...

Reply:
I kept is as 1.3.0 and will overwrite the package that was already published (for once)